### DISCUSSION TITLE
Addon for quests

### DISCUSSION THREAD

**Discussion by matlab00** [chromiecraft/chromiecraft#3621] [General]
Hi,
is there addon that shows all available not yet completed quests on the map?

**Comment by 55Honey** [comment]
Hi matlab00,

I suggest you join our discord community for detailed discussions about addons: https://discord.gg/Chromiecraft

A quick google search should yield some suggestions about this subject, too and you don't even need to wait for a reply.
See https://www.chromiecraft.com/en/downloads/ with a link to a repository with a lot of addons tailored for WoW 3.3.5.

Regards

**Reply by cachocarne** [reply]
You can use EveryQuest2, a simple addon that will show you all available quests by zone and the level required for it. However, those will not appear on the map, you will still have to do a little research when  you see you can't find it.

**Comment by Cooties-cc** [comment]
I use to use EveryQuest to scour for quests to do, but I found it a little clunky overall in how it tracked data. I then swapped over to using QuestCompletist addon, which is a little more streamlined. However it does have some problems itself, while the only working wotlk version I found was a cata patch backport from the [felbite website](https://felbite.com/addon/3458-questcompletist/) (I don't really like them). I then use Carbonite map addon to show questgivers and use the 2 to find what I am missing. 

If you want to use EveryQuest still, which is fine, I would suggest trying out using [EveryQuest QuestGivers](https://www.wowace.com/projects/everyquest-questgivers) in conjunction with it (if you don't want to use carbonite map addon). Afaik it's the easiest way to track down where you need to go from EQ data.
